ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/resources/jts/orbspecific/XAResourceRecord.java

```diff
@@ -120,7 +120,7 @@ public XAResourceRecord(TransactionImple tx, XAResource res, Xid xid,
 
 		if (params != null)
 		{
-			if (params.length >= XACONNECTION)
+			if (params.length >= XACONNECTION + 1)
```

Comment:
@mmusgrov I see, but I wanted to follow how is done here
https://github.com/jbosstm/narayana/blob/master/ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/transaction/jts/TransactionImple.java#L549

should I change the both places in way you suggest?